static/js/components/ProgressWidget_test.js

```diff
@@ -141,4 +142,21 @@ describe("ProgressWidget", () => {
     )
     assert.equal(wrapper.find(".circular-progress-widget-txt").text(), "3/5")
   })
+  it("progress widget display program certificate", () => {
```

Comment:
Can you change this text to `"should display program certificate when a certificate link exists"`